addons/binding/org.openhab.binding.autelis/ESH-INF/i18n/autelis_xx_XX.properties

```diff
@@ -0,0 +1,11 @@
+# binding
```

Comment:
You should remove this file - it is meant as an example on how to provide localized texts in other languages.